data/wp/wp-content/plugins/epfl/lib/model.php

```diff
@@ -177,7 +177,7 @@ function wp_post ()
      * handle that explicitly as well.)
      */
     function meta () {
-        if (! $this->_meta) {
+        if (! isset($this->_meta)) {
```

Comment:
Il ne faudrait pas plutôt utiliser `if (! property_exists($this, '_meta')) {` ?

Reply:
`isset` a le même «rendement» en matière de muselage des PHP Notices (contrôlé par mes soins) mais me paraît plus clair à la lecture. (Note : dans notre discussion ce midi, j'étais totalement à côté de la plaque en matière de contrôle de la «truthiness» du champ)